gui/widgets/richtext.cpp

```diff
@@ -32,6 +32,15 @@
 #include "gui/widgets/richtext.h"
 #include "gui/widgets/scrollbar.h"
 
+namespace Graphics {
```

Comment:
What is the reason of putting it in the Graphics namespace?

Reply:
fixed